src/hyperion/__main__.py

```diff
@@ -45,7 +46,7 @@ class Command:
     devices: Optional[Any] = None
     experiment: Optional[Callable[[Any, Any], MsgGenerator]] = None
     parameters: Optional[InternalParameters] = None
-    callbacks: Optional[type[AbstractPlanCallbackCollection]] = None
+    callbacks: Optional[Callable[[], Tuple[CallbackBase, CallbackBase]]] = None
```

Comment:
Could: This type is getting a bit unwieldy, maybe create do a:
```python
CallbackFactory = Callable[[], Tuple[CallbackBase, CallbackBase]]
```
somewhere and use that?